CHANGELOG.md

```diff
@@ -0,0 +1,27 @@
+# Changelog
+
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
+and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
+
+## [Unreleased]
+
+### Added
+
+- Check on tarantool box initialization before init (now it returns a nice error).
+- Posibility to pass port to `memcached.create()` as a number.
+
+### Changed
+
+- Documentation has been updated.
+- Update an upper value for `readahead` parameter (maximum value was taken from
+  validation code
+  https://github.com/tarantool/tarantool/blob/297c33492f8697350f78754fe3cb166f700c16ae/src/box/box.cc#L1091-L1100).
+- Switched to using the module's own slab arena. To get the information about slab arena
+  the `memcached.slab.info()` can be used.
```

Comment:
Inconsistent tense: imperative mood vs past simple. Let's use past simple. See https://www.tarantool.io/en/doc/latest/contributing/release_notes/

Reply:
I tried it, I don't know if I got it right.

Reply:
Updated.